exist-core/src/main/java/org/exist/xmlrpc/RpcConnection.java

```diff
@@ -1155,21 +1148,21 @@ private Map<String, List> listDocumentPermissions(final XmldbURI collUri) throws
     }
 
     @Override
-    public Map<XmldbURI, List> listCollectionPermissions(final String name)
+    public Map<String, List<Object>> listCollectionPermissions(final String name)
```

Comment:
I am interested to know the thinking behind why this should be changed from `XmldbURI` to `String` please?

Reply:
Presumably this was a new test that you added that was previously missing?

Reply:
Correct, among a lot of others 😎